src/app/public/modules/action-hub/action-hub.component.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { TestBed } from '@angular/core/testing';
+import { expect } from '@skyux-sdk/testing';
+
+import { SkyActionHubComponent } from './action-hub.component';
+import { SkyActionHubModule } from './action-hub.module';
+
+describe('Action hub component', async () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [SkyActionHubModule]
+    }).compileComponents();
```

Comment:
You don't necessarily need the `compileComponents` here but again not something to block this over.